.env

```diff
@@ -1,3 +1,5 @@
+DEBUG = 'true'
```

Comment:
I think it needs comments about possible values, because here it is string representation of boolean and if the administrator would change it to `False` it wouldn't work and it isn't intuitive

Reply:
Fixed